chart/templates/deployment.yaml

```
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: backend
 spec:
-  replicas: 1
+  replicas: 2
```

вроде ты говорил, что делал это чисто чтобы потестить локи, не вижу смыслов от двух инстансов нам)
```suggestion
  replicas: 1
```